server/routes.go

```diff
@@ -720,9 +720,49 @@ func GetModelInfo(req api.ShowRequest) (*api.ShowResponse, error) {
 	fmt.Fprint(&sb, model.String())
 	resp.Modelfile = sb.String()
 
+	ggmlData, err := getGGMLData(model)
+	if err != nil {
+		return nil, err
+	}
+	resp.ModelInfo = ggmlData
+
 	return resp, nil
 }
 
+func getGGMLData(model *Model) ([]byte, error) {
+	f, err := os.Open(model.ModelPath)
+	if err != nil {
+		return nil, err
+	}
+
+	ggml, _, err := llm.DecodeGGML(f)
+	if err != nil {
+		return nil, err
+	}
+
+	f.Close()
+
+	kv := ggml.KV()
+
+	for k := range kv {
+		switch v := kv[k].(type) {
+		case []interface{}:
+			if len(v) > 5 {
+				kv[k] = []string{}
+			}
+		}
```

Comment:
I mentioned this in the other PR but showing an empty list is very confusing. Instead, this should show how many items are in the list. TBH this doesn't even need a length check, just always output the count

```go
if t, ok := kv[k].([]any); ok {
    kv[k] = fmt.Sprintf("(%d items)", len(t))
}
```